x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curation_suggestion/curation_suggestion.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiSpacer,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { useDecodedParams } from '../../../../utils/encode_path_params';
+import { AppSearchPageTemplate } from '../../../layout';
+import { Result } from '../../../result';
+import { Result as ResultType } from '../../../result/types';
+import { getCurationsBreadcrumbs } from '../../utils';
+
+import { CurationActionBar } from './curation_action_bar';
+import { CurationResultPanel } from './curation_result_panel';
+
+import { DATA } from './temp_data';
+
+export const CurationSuggestion: React.FC = () => {
+  const { query } = useDecodedParams();
+  const [showOrganicResults, setShowOrganicResults] = useState(false);
+  const currentOrganicResults = [...DATA].splice(5, 4);
+  const proposedOrganicResults = [...DATA].splice(2, 4);
+
+  const queryTitle = query === '""' ? query : `${query}`;
+
+  return (
+    <AppSearchPageTemplate
+      pageChrome={getCurationsBreadcrumbs([
+        i18n.translate(
+          'xpack.enterpriseSearch.appSearch.engine.curations.suggestedCuration.breadcrumbLabel',
+          { defaultMessage: 'Suggested: {query}', values: { query } }
+        ),
+      ])}
+      pageHeader={{
+        pageTitle: queryTitle,
+      }}
+    >
+      <CurationActionBar
+        onAcceptClick={() => alert('Accepted')}
```

Comment:
FWIW, the interface of some of these components will likely change when I wire it up.

For instance, instead of passing in handlers, this component will probably call kea actions directly and take no props.